content/en/about/governance.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Governance and Decision Making"
+draft: false
+---
+
+# Abstract
+
+Scientific Python is a consensus-based community project.
+Anyone with an interest in the project can join the community, contribute to the project design,
+and participate in the decision making process.
+This document describes how that participation takes place, how to find consensus, and how
+deadlocks are resolved.
+
+## Roles And Responsibilities
+
+### Community Members
+
+The Scientific Python community consists of anyone working with the project in any way.
+Community members must adhere to our [code of conduct]({{< relref "/code_of_conduct.md" >}}).
+
+### Community Managers
+
+Community Managers are community members that have demonstrated continued
+commitment to the project through ongoing contributions.
+
+New Community Managers can be nominated by any existing Community Manager.
+Discussion about new Community Manager nominations is one of the few
+activities that takes place on the project\'s private management list.
+The decision to invite a new Community Manager must be made by "lazy consensus",
+meaning unanimous agreement by all responding existing Community Manager.
+Invitation must take place at least one week after initial nomination,
+to allow existing members time to voice any objections.
+
+### Leadership Team
+
+The Leadership Team (LT) members are community managers who have additional responsibilities to ensure the smooth running of the project.
+LT members are expected to participate in strategic planning, approve changes to the governance model, and make decisions about funding granted to the project itself.
+(Funding to community members is theirs to pursue and manage.)
+The purpose of the LT is to ensure smooth progress from the big-picture perspective.
+Changes that impact the full project require analysis informed by long experience with both the project and the larger ecosystem.
+When the core developer community (including the LT members) fails to reach such a consensus in a reasonable timeframe, the LT is the entity that resolves the issue.
```

Comment:
```suggestion
When the core developer community (including the LT members) fail to reach such a consensus in a reasonable timeframe, the LT is the entity that resolves the issue.
```

Reply:
I think we typically use plural when members are in disagreement, i.e. operate as many, but can go either way.